src/WebJobs.Script.WebHost/ContainerManagement/LinuxContainerActivityPublisher.cs

```diff
@@ -165,7 +165,7 @@ private async Task FlushFunctionExecutionActivities()
             }
             catch (Exception exc) when (!exc.IsFatal())
             {
-                _logger.LogError(exc, $"{nameof(FlushFunctionExecutionActivities)}");
+                _logger.LogError(exc, nameof(FlushFunctionExecutionActivities));
```

Comment:
😕

Reply:
Is it okay to change/improve these? Thus far I've been very wary of breaking changes, but happy to help improve the errors (I haven't yet learned which ones customers see vs. not)